apps/pdc-frontend/src/util/generateSurveyURL.ts

```diff
@@ -0,0 +1,99 @@
+export interface SurveyRunParams {
+  guid?: string;
+  view?: string;
+  apikey?: string;
+  istestresult?: string;
+  locale?: string;
+  segment?: string;
+  frontendPublicURL?: string;
+  surveyURL?: URL;
+}
+
+export const getPublicURL = (env: any) => {
+  try {
+    if (!env.FRONTEND_PUBLIC_URL) {
+      if (
+        process.env['NODE_ENV'] === 'development' ||
+        process.env['NODE_ENV'] === 'test' ||
+        process.env['DEBUG'] === 'true'
+      )
+        throw new Error('FRONTEND_PUBLIC_URL is not defined');
+      return undefined;
+    }
+    return new URL(env.FRONTEND_PUBLIC_URL).origin;
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error('Error getting public URL:', error);
```

Comment:
Ik denk dat deze melding in de logs niet duidelijk genoeg is om 'm te kunnen fixen. Ik zou in elk geval `FRONTEND_PUBLIC_URL` noemen. Bijvoorbeeld:
`No valid URL found in the required environment variable FRONTEND_PUBLIC_URL`